PhysicsTools/NanoAOD/python/jets_cff.py

```diff
@@ -253,7 +255,15 @@
 jetTable.variables.pt.precision=10
 
 ### Era dependent customization
-run2_jme_2016.toModify( jetTable.variables, jetId = Var("userInt('tightIdLepVeto')*4+userInt('tightId')*2+userInt('looseId')",int,doc="Jet ID flags bit1 is loose, bit2 is tight, bit3 is tightLepVeto"))
```

Comment:
this PR as now at commit 39aa5df
change the behaviour of the 2016 jetID for the run2_nanoAOD_106Xv1, I can see also from the DQM plot.
restoring the behaviour of master.

Reply:
@mariadalfonso no changes are required then?

Reply:
looks like this line is wrong in 10_6_X.
It should be modified like in the master:
https://github.com/cms-sw/cmssw/blob/master/PhysicsTools/NanoAOD/python/jets_cff.py#L271
for modifier in run2_miniAOD_80XLegacy, run2_nanoAOD_94X2016:
  modifier.toModify( jetTable.variables, jetId = Var("userInt('tightIdLepVeto')*4+userInt('tightId')*2+userInt('looseId')",int,doc="Jet ID flags bit1 is loose, bit2 is tight, bit3 is tightLepVeto"))
Would you mind fixing this here?

Reply:
@ahinzmann thanks for the quick feedback
the fix is done with this PR at commit 39aa5df restore the behaviour of master for this

@andrzejnovak
please add in the PR description a comment on this, together with Slava's requests